packages/core-sdk/lib/Types/IConnector.ts

```diff
@@ -9,14 +9,21 @@ export interface IConnectorAdapters {
   multicall: IMulticallAdapter;
 }
 
+export interface OnNetworkChangeOptions {
+  tryReconnection: boolean;
```

Comment:
I think it has more sense to be called "tryChangingNetwork" because the name "tryReconnection" sounds like it will try to reconnect on any kind of error when the only thing it will do is to try to change the network